src/makielayout/blocks/axis.jl

```diff
@@ -919,9 +919,12 @@ end
 """
     autolimits!(la::Axis)
 
-Reset manually specified limits of `la` to an automatically determined rectangle, that depends on the data limits of all plot objects in the axis, as well as the autolimit margins for x and y axis.
+Reset manually specified limits of `la` to an automatically determined rectangle, that depends on the 
```

Comment:
while we're at it, you can change `la` to `ax` here, `la` is still from `LAxis` times